modules/channel-web/src/hooks/after_incoming_middleware/00_dialog_engine.js

```diff
@@ -1,5 +1,30 @@
-const messageTypesToDiscard = ['request_start_conversation']
+const messageTypesToDiscard = ['request_start_conversation', 'say_something', 'postback']
 
 if (messageTypesToDiscard.includes(event.type)) {
   event.setFlag(bp.IO.WellKnownFlags.SKIP_DIALOG_ENGINE, true)
+
+  if (event.type === 'say_something' && event.payload.text) {
+    saySomething()
```

Comment:
is `await` missing on purpose?  if yes please add a comment because I'd be tempter to add it in the future if I came across it

Reply:
No particular reason other than the build fails since its 'parent' is not async

![image](https://user-images.githubusercontent.com/42552874/55555781-0df87700-56b4-11e9-9616-be35585463e9.png)